pallets/orml-currencies-allowance-extension/Cargo.toml

```diff
@@ -26,27 +26,27 @@ pallet-balances = { git = "https://github.com/paritytech/polkadot-sdk", branch =
 
 [dev-dependencies]
 mocktopus = "0.8.0"
-frame-benchmarking = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0"}
+frame-benchmarking = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
 sp-io = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false }
 
-pallet-balances = {git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0"}
+pallet-balances = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
 
 
 [features]
 default = ["std"]
 std = [
-  "serde/std",
-  "codec/std",
-  "sha2/std",
-  "sp-core/std",
-  "sp-std/std",
-  "sp-runtime/std",
-  "frame-support/std",
-  "frame-system/std",
-  "orml-currencies/std",
-  "orml-tokens/std",
-  "orml-traits/std",
-  "frame-benchmarking/std"
+    "serde/std",
```

Comment:
Is this caused by cargo fmt?

Reply:
No, I think RustRover automatically formatted it like this 😬 A bit annoying that this is always changed back and forth but IIRC we already looked into an automatic formatting solution some time ago and nothing convenient was available. I think rustfmt doesn't support .toml files.